specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json

```diff
@@ -532,6 +565,113 @@
           }
         }
       }
+    },
+    "ManagedRuleSets": {
+      "description": "Defines ManagedRuleSets - array of managedRuleSet",
+      "properties": {
+        "ruleSets": {
+          "description": "List of rules",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedRuleSet"
+          }
+        }
+      }
+    },
+    "ManagedRuleSet": {
+      "type": "object",
+      "description": "Base class for all types of ManagedRuleSet.",
+      "x-ms-discriminator-value": "Unknown",
+      "properties": {
+        "ruleSetVersion": {
+          "description": "defines version of the rule set",
```

Comment:
Please update to have first letter capitalized and dot in the end (for all the descriptions). Feel free to reuse any commits or create patch from [there](https://github.com/Azure/azure-rest-api-specs/pull/5965).
You could find more info on this [there](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-authoring-descriptions.md#basic-rules-for-public-docs-descriptions).

Currently NetworkValidation step of CI already checks if the first letter capitalized and later we are going to introduce more checks.

Reply:
Done